docs-chef-io/content/inspec/cloud_security.md

```diff
@@ -0,0 +1,212 @@
+
+### Benefits of Chef's Cloud Security
+
+- Built-in compliance that maintains continuous compliance across the entire cloud estate.
+
+- Coordination of security, development, testing, and operations participation to make it easier to carry out security checks at every stage of the DevOps pipeline with coded artifacts, automated tests and enterprise control, and visibility.
+
+- Continuously monitors cloud environments in real-time for threat detection;
+
+- Detects misconfigurations and aids in prioritizing them across clouds.
+
+- Detects common Identity and Access Management (IAM) risks and help prioritizes misconfigurations across the public cloud to meet IAM compliance mandates- passwords, key rotation, and so on.
+
+- Facilitation of existing skillsets without time-consuming and costly training for managing compliance and security posture for cloud-native technologies, including containers, Kubernetes, and microservices.
+
+- Functions with IaaS, SaaS, and PaaS platforms in containerized, hybrid cloud, and multi-cloud environments.
+
+- Maintains an inventory of best practices for different cloud configurations and services.
+
+- Maps current configuration statuses to a security control framework or regulatory standard.
+
+- Monitors storage buckets, encryption, and account permissions for misconfigurations and compliance risks.
+
+- Support for native-cloud assets for multi-cloud deployment across AWS, Azure, Google, and Alibaba clouds using certified CIS/DISA STIG profiles to scan and detect security configuration issues.
+
+- Support SOC2 (Security Operation Center) compliance, voluntary compliance standard for service organizations, for cloud accounts.
+
+### Chef CSPM Framework
+
+The key features of the CSPM tools include the ability to:
+
+- End-to-end visibility governance
+  - Robust Detections: Easy to read, filterable scan reports (control summary) with detailed descriptions of all security controls and their results.
+  - Multi-Cloud Reporting: Cloud Integration support - GCP, AWS, Azure
+  - Data Aggregation: Data Export into 3rd Party tools (ServiceNow, Splunk)
+  - Azure: DevOps Governance and Policy Integration without Automate
+  - Third-party tools: Terraform, Postgres
+
+- Containers
+  - Best practices for Docker Enterprise Edition based on CIS
+
+- K8 Pods
+  - Security profiles for Kubernetes based on CIS benchmarks
+  - Remediation content for Kubernetes environment for security configuration violations based on CIS standards
+
+- Cloud
+  - Pre-built service and resource templates for custom policies across 100+ AWS, GCP, and Azure resources
+  - Policy Compliance for the latest OS Versions (CentOS, RHEL, Ubuntu) & DB versions (Oracle, PostgreSQL)  as per CIS best practices.
+
+- Host
+  - Remediation Content: Accessing Remediate profiles from Chef UI
+  - Fast, Scalable Remediation for security configuration violations based on CIS standards
```

Comment:
This is just a big wall of text. Can we split this up with some introductory sentences for each section.